rdsbroker/broker.go

```diff
@@ -166,6 +175,14 @@ func (b *RDSBroker) Update(instanceID string, details brokerapi.UpdateDetails, a
 	}
 
 	modifyDBInstance := b.modifyDBInstance(instanceID, servicePlan, updateParameters, details)
+
+	if servicePlan.RDSProperties.ConfigureAllocatedStorage && updateParameters.AllocatedStorage > 0 {
```

Comment:
Isn't `if servicePlan.RDSProperties.ConfigureAllocatedStorage` code redundant with the one in the `validateStorage` method?